packages/charts/src/charts/metric/metric-chart.styles.ts

```diff
@@ -1,11 +1,26 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
-export const Layout = styled.div`
-  position: relative;
-  display: flex;
-  height: 100%;
-  justify-content: space-between;
-  flex-direction: row;
+export type IconPosition = 'top' | 'center' | 'bottom';
+
+export type Margins = {
```

Comment:
Obsolete type - which conflicts with global `Margins` type . Additionally the margins are not working correctly for the icon.